cache-external-data/action.yml

```diff
@@ -0,0 +1,19 @@
+name: "Cache CMake External Data"
+description: "Caches the external data that we fetch via CMake"
+inputs:
+  store-key-base:
+    required: False
+    description: The base of the store key (will be suffixed with todays date)
+    default: "external-data"
```

Comment:
I'm not sure this is needed since it won't be used but it doesn't hurt either

Reply:
We have it in some cases, and I think it's still quicker to fetch it from the gh cache then going via https to EOS.

Reply:
Ah right. I put it there, just in case someone wants to put it somewhere else. It's not strictly necessary, but as you said it doesn't hurt and at this point also doesn't cost us anything.